doc/classes/VisualInstance.xml

```diff
@@ -62,6 +62,16 @@
 			The render layer(s) this [VisualInstance] is drawn on.
 			This object will only be visible for [Camera]s whose cull mask includes the render object this [VisualInstance] is set to.
 		</member>
+		<member name="sorting_offset" type="float" setter="set_sorting_offset" getter="get_sorting_offset" default="0.0">
+			The sorting offset used by this VisualInstance.
+			A higher sorting offset will make the VisualInstance draw on top of other [VisualInstance] that are otherwise positioned at the same spot.
+			Best used with [member sorting_use_aabb_center].
+		</member>
+		<member name="sorting_use_aabb_center" type="bool" setter="set_sorting_use_aabb_center" getter="is_sorting_use_aabb_center" default="true">
+			Use the AABB center for transparent object sorting.
+			Best turned off when using [member sorting_offset].
```

Comment:
The docs for `sorting_offset` say "Best used with [member sorting_use_aabb_center].", and this says to turn the other option off. So which is it?